test/validators.js

```diff
@@ -2599,6 +2599,33 @@ describe('Validators', () => {
     });
   });
 
+  it('should validate HSL color strings', () => {
+    test({
+      validator: 'isHSL',
+      valid: [
+        'hsl(360,0000000000100%,000000100%)',
+        'hsl(000010, 00000000001%, 00000040%)',
+        'HSL(00000,0000000000100%,000000100%)',
+        'hsL(0, 0%, 0%)',
+        'hSl(  360  , 100%  , 100%   )',
```

Comment:
Thanks for this PR. 
Here are some of the notes from my research on `hue`

> H (hue) is an <angle> of the color circle given in degs, rads, grads, or turns in CSS Color Module Level 4. When written as a unitless <number>, it is interpreted as degrees, as specified in CSS Color Module Level 3. By definition, red=0deg=360deg, with the other colors spread around the circle, so green=120deg, blue=240deg, etc. As an <angle>, it implicitly wraps around such that -120deg=240deg, 480deg=120deg, -1turn=1turn, etc.

Thus, can we try to catch some of the edge cases(valid cases), e.g 
```
hsl(4.71239rad, 60%, 70%)
hsl(270deg, 60%, 70%)
hsl(270 60% 70%)
hsl(.75turn, 60%, 70%)
``` 

Last, we can have a discussion on:

> ***TBD*** ---->>> HSL colors are expressed through the functional hsl() and hsla() notations. E.g we can introduce isHSLA() or incorporate it in here 

For clarification or assistance on the following refer to [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value)

Reply:
The hsla has an extra parameter, alpha, which can hold values between [1,0]. Therefore, it should have its own validator function.

Reply:
Thanks, @guischafer.

Reply:
Thank you all for the feedback and links to documentation. A bit busy at the moment, but I think I can make the appropriate changes before next weekend.

Reply:
The above edge cases are now handled. The optional 4th parameter (alpha) is also handled as `hsl()` supports it (`hsla()` is just an alias to `hsl()`).

Reply:
Good one @kurisu-gh. That means that it will close #1245